ci-operator/step-registry/openstack/provision/securitygroups/openstack-provision-securitygroups-commands.sh

```diff
@@ -29,7 +29,11 @@ if [[ -n "$ADDITIONAL_SECURITY_GROUP_RULES" ]]; then
 			case $service in
 				netperf)
 					echo "Adding ${service} rule to security group ${sg_id}" 
-					openstack security group rule create "$sg_id" --protocol tcp --dst-port 12865:12865 --remote-ip 0.0.0.0/0
+					openstack security group rule create "$sg_id" --protocol tcp --dst-port 12865:12865 --remote-ip 0.0.0.0/0 --description netperf
+					openstack security group rule create "$sg_id" --protocol tcp --dst-port 22865:22865 --remote-ip 0.0.0.0/0 --description iperf3
+					openstack security group rule create "$sg_id" --protocol tcp --dst-port 30000:30000 --remote-ip 0.0.0.0/0 --description uperf
+					openstack security group rule create "$sg_id" --protocol tcp --dst-port 32000:47000 --remote-ip 0.0.0.0/0 --description netserver-tcp
```

Comment:
Would you have links for these port ranges? they're pretty large, is there a way to make them smaller?

Reply:
Also, if we know the IP range from where this connection is supposed to initiate, we can restrict the source IP, and stop using 0.0.0.0/0.

Reply:
It works if I set the `machineNetwork` to `netserver-tcp` and `netserver-udp` rules. But at this point `openstack-provision-securitygroups` we don't know what is going to be the `machineNetwork`. By default, if it runs the `minimal` type, it will be the 10.0.0.0/16 due to openshift-install will create a machine subnet with that CIDR.